models/us_census_acs/code/upload.py

```
@@ -0,0 +1,119 @@
+    "data_profile_zcta",
+    "microdata_household",
+    "microdata_person",
+]
+
+
+def local_rows(path: Path) -> int:
+    try:
+        return pads.dataset(
+            str(path), format="parquet", partitioning="hive"
+        ).count_rows()
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)